### DISCUSSION TITLE
How to use --resource-path in quarto docs?

### DISCUSSION THREAD

**Discussion by sje30** [quarto-dev/quarto-cli#5080] [Q&A]
I'm trying to use --resource-path to point to a different folder for where I store images.

With the following simple example:

````
---
title: "test 1"
resource-path: ["/tmp"]
format:
  html:
    embed-resources: true
---
# Generate a plot

```{r}
png(file="/tmp/temp.png")
plot(1:10)
dev.off()
```


# show the plot v1

![alt text](/tmp/temp.png "tooltip")

# show the plot v2

![alt text](temp.png "tooltip")
````

when I render in quarto, v1 of the plot appears, but not v2.

    quarto render temp.qmd

However, if I run pandoc from the command line, it works (as in the plot appears twice):

    pandoc --resource-path='/tmp' --self-contained -s -o temp.html temp.qmd

what am I missing?

Thanks, Stephen

```
pandoc v 3.1.1
quarto  v 1.2.475
running on Mac
```

**Comment by cderv** [comment]
Thanks for the repot. I'll move this to an issue, as I think we need to handle `resource-path` better in our process.

**Comment by ghost** [comment]
Moved to  #5084
